tests/utils_tests/test_metaclassmaker.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```

Comment:
the coding line is normally only included if there are non-ascii characters in the file